source/docs/articles/drupal/adding-yml-file.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Adding a services.yml File
+description: Learn how to add a services.yml file for Drupal 8.
+category:
+  - developing
+  - drupal
+keywords: drupal, services.yml, yml file
+---
+With Drupal 8, you may receive a "permission denied" error when trying to add a services.yml file under the `sites/default` folder.
+
+This happens when you are trying to add it in while you're in SFTP mode. You need to switch to Git mode to add the file. This is because the `sites/default` directory has the permissions of 555, even if you toggle SFTP mode on and off.
+
+## Add a services.yml File in Git
+
+1. On the Code tab of your Dashboard, click **Git**.  
+2. Create the file locally.  
+3. Open your terminal and run `git add .`.  
+4. Run `commit -m"enter a commit message`.  
+5. Push the file by running `git push origin origin-name`.
```

Comment:
We should include the correct permissions for the `sites/default` folder so this file can be created via SFTP, suggested in the [related issue](https://github.com/pantheon-systems/drops-8/issues/26#issuecomment-78310615)

The correct permissions for the `sites/default` directory is 555 (read-execute) - which can be set via Git by running `chmod a+w sites/default` or from within an SFTP client.

[Step 3: Create settings.php, services.yml and the files directory](https://www.drupal.org/documentation/install/settings-file) has some good info we can use. @bmackinney should we broaden the scope of this doc to cover more installation topics?